strapi/strapi
# Fix missing review workflows column

### What does it do?

- Fixes a missing column for review workflows in the ListView

### Why is it needed?

It is a regression from 4.11.2

### How to test it?

Enable review workflows on a model
Create some entries
You should see the review workflow stage column in the ListView table
